smart-server/pom.xml

```diff
@@ -33,6 +33,31 @@
 
     <dependencies>
         <dependency>
+            <groupId>com.sun.jersey</groupId>
```

Comment:
First declare dependencies in root pom.xml's dependency management segment, than add dependency in smart-server/pom.xml